src/organizations/components/ComponentsOrganizationCollectionEntity.js

```diff
@@ -0,0 +1,10 @@
+import { OrganizationCollectionEntity } from '../OrganizationCollectionEntity.js';
+
+/*
+ * A collection of organization components to implement specific functionality.
+ */
+export class ComponentsOrganizationCollectionEntity extends OrganizationCollectionEntity {
```

Comment:
Should this be inheriting from `Entity` instead of `OrganizationCollectionEntity`?

Reply:
🤔

Reply:
Fair enough 👍